plugins/outputs/nats/nats.go

```diff
@@ -85,7 +100,47 @@ func (n *NATS) Connect() error {
 
 	// try and connect
 	n.conn, err = nats.Connect(strings.Join(n.Servers, ","), opts...)
+	if n.Jetstream != nil {
+		// connect to jetstream
+		n.jetstreamClient, err = jetstream.New(n.conn)
+		if err != nil {
+			return fmt.Errorf("failed to connect to jetstream: %w", err)
+		}
+		if n.Jetstream.Stream == "" {
+			return errors.New("stream cannot be empty")
+		}
+		streamExists := n.streamExists(n.Jetstream.Stream)
```

Comment:
Let's check the value immediately and return to avoid a lot of indents.

```suggestion
		if n.streamExists(n.Jetstream.Stream) {
		   return nil
		}
```